src/equations/hyperbolic_diffusion_2d.jl

```diff
@@ -52,6 +52,14 @@ end
   return SVector(phi, q1, q2)
 end
 
+@inline function initial_condition_peak(x, t, equations::HyperbolicDiffusionEquations2D)
+  # elliptic equation: -ν Δϕ = f in Ω, u = g on ∂Ω
+  phi = exp(-x[1]^2 - (x[2] - 0.5)^2)
+  q1  = exp(-x[1]^2 - (x[2] - 0.5)^2)
+  q2  = exp(-x[1]^2 - (x[2] - 0.5)^2)
+  return SVector(phi, q1, q2)
+end
+
```

Comment:
```suggestion
```
I don't think we need this functionality in this reduced PR

Reply:
Done.